src/Symfony/Component/Webhook/Controller/WebhookController.php

```diff
@@ -40,12 +40,17 @@ public function handle(string $type, Request $request): Response
         }
         /** @var RequestParserInterface $parser */
         $parser = $this->parsers[$type]['parser'];
+        $events = $parser->parse($request, $this->parsers[$type]['secret']);
 
-        if (!$event = $parser->parse($request, $this->parsers[$type]['secret'])) {
+        if (!$events) {
             return $parser->createRejectedResponse('Unable to parse the webhook payload.', $request);
         }
 
-        $this->bus->dispatch(new ConsumeRemoteEventMessage($type, $event));
+        $events = \is_array($events) ? $events : [$events];
+
+        foreach ($events as $event) {
```

Comment:
Can be optimized to:
```suggestion
        foreach ((array) $events as $event) {
```

Reply:
When not an array, this would cast the object to an array, no?

Reply:
Ah, you're right

Reply:
be careful. For that particular case it works, but if the object can be an object implementing `\Traversable`, it won't be equivalent